app/models/user.rb

```diff
@@ -4,5 +4,17 @@ class User < ApplicationRecord
   devise :invitable, :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  validates :full_name, presence: true, uniqueness: true
+  validates :full_name, presence: true
+
+  def admin?
+    role == "admin"
+  end
+
+  def advanced?
+    role == "advanced"
+  end
+
+  def editor?
+    role == "editor"
```

Comment:
kinda feel like we should keep the terminology that the service designers use i.e. "basic" rather than "editor"